tests/benchmarks/test_futures.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+from dask.distributed import as_completed, wait
+from distributed.utils_test import inc, slowdec, slowinc
+
+
+def test_single_future(small_client):
+    """How quickly can we run a simple computation?"""
+    small_client.submit(inc, 1).result()
```

Comment:
This will capture benchmarking information, but I'm not sure I would trust any of it, as I'd expect it to be 100% background noise.

Reply:
🤔 what are the sources of background noise these days?  Is it test setup, or is it Dask/Coiled runtime noise?

Reply:
I'm tempted now to make a test that is just ...


```python
def test_foo(small_client):
    pass
```

Reply:
We've got a similar test already 

https://github.com/coiled/coiled-runtime/blob/c11ea3a6a41ae4d88eef921c14e8de5ec9f04040/tests/runtime/test_cluster_creation.py#L6-L15

Reply:
That test makes a new cluster.  I'm curious about noise when we're using the existing cluster from the fixture (I would assume that fixture creation time doesn't get counted in metrics).  

I was mostly joking though.  Mostly.  It sounds like noise is a concern.  We should maybe be intentionally sensitive to noise.